script/benchmark

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly BUNDLE_GEMFILE="$(pwd)/tmp/bench-site/Gemfile" bundle update
+
+function fold() {
+  echo "travis_fold:start:$1"
+  $@
+  echo "travis_fold:end:$1"
+}
+
+function benchmark() {(
+  cd tmp/bench-site
+  fold bundle update
+  echo "$1" > "../$1.txt"
+  start=$(date +%s)
+  bundle exec jekyll build
+  end=$(date +%s)
+  echo "$((end-start))" >> "../$1.txt"
+)}
+
+function gemfile() {
+  cat > tmp/bench-site/Gemfile << EOF
+  source "https://rubygems.org"
+  gem "jekyll", $@
+  gem "minima"
+EOF
+}
+
+function setup() {
+  rm -rf tmp/benchmark tmp/bench-site
+  mkdir -p tmp/bench-site
+  git clone --depth 1 https://github.com/jekyll/benchmarking.git tmp/benchmark
+}
+
+function generate_site() {
+  tmp/benchmark/generate_site.sh 10000 tmp/bench-site
+}
+
+function display() {
+  THIS=$(tail -n 1 tmp/this.txt)
+  MASTER=$(tail -n 1 tmp/this.txt)
+  DIFF=$(echo "scale=4;($THIS-$MASTER)/($MASTER/100)" | bc)
+  pr -m -t tmp/this.txt tmp/master.txt
+  echo "This is $DIFF% the speed of master"
+  echo "100% means no performance change, lower than 100% means this is faster."
+}
+
+fold setup
+fold generate_site
+
+gemfile ":path => '../..'"
+benchmark this
+
+gemfile ":github => 'jekyll/jekyll', branch: 'master'"
```

Comment:
I prefer using `:git => 'https://github.com/jekyll/jekyll.git',  branch: 'master' to avoid seeing warnings from Bundler regarding the use of a non-secure protocol to fetch the commit

Reply:
I think I must have turned off that warning on my local machine 😜
Sure, I can change this :+1: